test/rules/new-line-before-expect.js

```diff
@@ -49,6 +49,13 @@ eslintTester.run('new line before expect', rule, {
       '}));'
     ]),
     linesToCode([
+      'it("", helper(function() {',
+      '  var a = 1',
+      '',
+      '  expect(a).toBe(1); expect(a).not.toBe(0);',
```

Comment:
yeap, I agree this rule shouldn't care if there are multiple expects on the line.